pkg/session/session.go

```diff
@@ -195,17 +200,21 @@ func (s *Session) keepAlive() {
 			writerExit := make(chan any)
 			s.closer = &closeOnce{closer: make(chan any)}
 			ctx, cancel := context.WithCancel(context.Background()) // create local context for each session
-			jar, _ := cookiejar.New(nil)
 
 			log.Logger.Infow("session keep alive: checking server health")
-			if err := s.checkServerHealth(ctx, jar); err != nil {
+			if err := s.checkServerHealth(ctx); err != nil {
```

Comment:
The health check should be done with sessionAddr, otherwise the check is meaningless

Reply:
#817 changed endpoint in checkServerHealth to local gpud addr, I thought it is expected

Reply:
currently we can fully remove this function, shall we do it now?